scripts/aof.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
Rename script to clarify that this is a test, not (for example) a recovery tool. (e.g. `test-aof.sh`).